File: test/test-question-do-loop.js
const gb = require("./gbtest")(__filename);

test("?do...loop", () => {
  gb.run();
  expect(gb.stack).toEqual([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 99, 100, 101]);
});

File: test/test-question-do-loop.fs
: test-1
  10 0 ?do i loop 99 ;

: test-2
  5 0 ?do
    i
    i 2 = if unloop exit then
  loop
  99 ;

: test-3
  5 0 ?do
    i
    i 2 = if leave then
  loop
  99 ;

: main
  test-1
  test-2
  test-3
;
